### ISSUE TITLE
Multiple room at the same time

### ISSUE THREAD

**Issue by OldLions** [vahidhedayati/grails-wschat-plugin#7]
Hi,
Is there a way to have an user logged into multiple rooms at the same time?

When I try, the following message appears: `"${username} is already loggged in elsewhere, action denied"`.

Thank you!

**Comment by vahidhedayati** [comment]
Hi OldLions

Not at the moment, something that has been requested in the past.

Know what needs to be done, just not had the time to implement. Will take a look soon and update this when done.

**Comment by vahidhedayati** [comment]
I have got something working, something I was working on till the late hours last night. 

You didn't specify what Grails version ?

I have rolled out 1.20 for Grails 2.X and did some basic tests which appears to work. 

In short a user can log in multiple times now and they can only go to a room their not already in. If they attempt to switch rooms to a room their already in - again same rule applies they will be rejected access to that room.

No idea what else has broken due to this update. A lot of basic features tested and appeared to work. PM's sent from one user to a user logged in to multiple rooms appeared to hit the right window.

I have a feeling Friend PM may be broken in certain circumstances that Friend and user are not in the same room. 

Needs further testing.

Let me know if that works for you and if any issues again let me know I will look at porting it over to grails 3 soon
